api/user.go

```diff
@@ -36,8 +36,8 @@ type ModifySelfData struct {
 	AuditLogReason `json:"-"`
 }
 
-// ModifyMe modifies the requester's user account settings.
-func (c *Client) ModifyMe(data ModifySelfData) (*discord.User, error) {
+// ModifyCurrentUser modifies the requester's user account settings.
+func (c *Client) ModifyCurrentUser(data ModifySelfData) (*discord.User, error) {
```

Comment:
`ModifySelfData` should also be renamed to `ModifyCurrentUserData`.

Reply:
woops, forgot about that!